app/routers/grade.py

```
@@ -0,0 +1,49 @@
+
+
+@router.get(
+    "/{answersheet_id}/grade",
+    response_model=GradingResultResponse,
+    status_code=status.HTTP_200_OK,
+)
+async def get_grading_results_api(
+    answersheet_id: int,
+    page: int = Query(1, ge=1),  # 기본값은 1페이지부터 시작
+    page_size: int = Query(6, ge=1),  # 기본값은 한 페이지에 6개 표시
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)